build/dependencies.props

```diff
@@ -4,7 +4,7 @@
     <GoogleProtobufPackageVersion>3.12.2</GoogleProtobufPackageVersion>
     <GrpcDotNetPackageVersion>2.29.0</GrpcDotNetPackageVersion>  <!-- Used by example projects -->
     <GrpcPackageVersion>2.30.0-pre1</GrpcPackageVersion>
-    <MicrosoftAspNetCorePackageVersion>3.1.3</MicrosoftAspNetCorePackageVersion>
+    <MicrosoftAspNetCorePackageAppVersion>3.1.3</MicrosoftAspNetCorePackageAppVersion>
```

Comment:
PackageAppVersion is kinda strange. I know it was changed to match benchmarks but I'm curious why the naming scheme is used there.